MediaBrowser.Providers/MediaInfo/FFProbeProvider.cs

```diff
@@ -111,6 +116,15 @@ public bool HasChanged(BaseItem item, IDirectoryService directoryService)
                 return true;
             }
 
+            AudioResolver audioResolver = new AudioResolver();
+            if (item.SupportsLocalMetadata && video != null && !video.IsPlaceHolder
+                && !video.AudioFiles.SequenceEqual(
+                        audioResolver.GetExternalAudioFiles(video, directoryService, _namingOptions, false), StringComparer.Ordinal))
+            {
+                _logger.LogDebug("Refreshing {0} due to external audio change.", item.Path);
```

Comment:
Always use meaningful names in the log format strings
```suggestion
                _logger.LogDebug("Refreshing {ItemPath} due to external audio change.", item.Path);
```

Reply:
Sorry, but I literally copied the debug line from [a few lines above](https://github.com/jellyfin/jellyfin/blob/4ac2e24907e25b07979d76e571d73e102c1348e3/MediaBrowser.Providers/MediaInfo/FFProbeProvider.cs#L115). Should I change it nonetheless or change it for every debug line in this method? :)

Reply:
It's only important that new code follows the standards. If there's some quick fixes in the surrounding code that's ok, but PRs can quickly balloon in size with unrelated changes.